parsec/_parsec/__init__.pyi

```diff
@@ -255,3 +255,25 @@ class ChunkID:
     def from_bytes(bytes: bytes) -> ChunkID: ...
     def from_hex(hex: str) -> ChunkID: ...
     def new() -> ChunkID: ...
+
+from parsec._parsec.invite import (
+    InvitationToken,
+    SASCode,
+    generate_sas_code_candidates,
+    generate_sas_codes,
+    InviteUserConfirmation,
+    InviteDeviceData,
+    InviteDeviceConfirmation,
+    InviteUserData,
+)
+
+__all__ = [
```

Comment:
given the typing only contain exposed item, there is no need for `__all__` that can be omitted (beside this `__all__` is currently missing the other exposed stuff in __init__.pyi anyway)

Reply:
`mypy` give errors if I remove `__all__` 🥲

```patch
pre-commit run --all-files --show-diff mypy

mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

parsec/api/data/invite.py:3: error: Module "parsec._parsec" has no attribute "SASCode"  [attr-defined]
parsec/api/data/invite.py:3: error: Module "parsec._parsec" has no attribute "generate_sas_codes"; maybe "generate_sas_code_candidates"?  [attr-defined]
parsec/api/data/invite.py:3: error: Module "parsec._parsec" has no attribute "generate_sas_code_candidates"; maybe "generate_sas_codes"?  [attr-defined]
parsec/api/data/invite.py:3: error: Module "parsec._parsec" has no attribute "InviteUserData"  [attr-defined]
parsec/api/data/invite.py:3: error: Module "parsec._parsec" has no attribute "InviteUserConfirmation"; maybe "InviteDeviceConfirmation"?  [attr-defined]
parsec/api/data/invite.py:3: error: Module "parsec._parsec" has no attribute "InviteDeviceData"  [attr-defined]
parsec/api/data/invite.py:3: error: Module "parsec._parsec" has no attribute "InviteDeviceConfirmation"; maybe "InviteUserConfirmation"?  [attr-defined]
Found 7 errors in 1 file (checked 521 source files)

pre-commit hook(s) made changes.
If you are seeing this message in CI, reproduce locally with: `pre-commit run --all-files`.
To run `pre-commit` as part of git workflow, use `pre-commit install`.
All changes made by hooks:
diff --git a/parsec/_parsec/__init__.pyi b/parsec/_parsec/__init__.pyi
index 6f94062f4..9c7326d0b 100644
--- a/parsec/_parsec/__init__.pyi
+++ b/parsec/_parsec/__init__.pyi
@@ -266,14 +266,3 @@ from parsec._parsec.invite import (
     InviteDeviceConfirmation,
     InviteUserData,
 )
-
-__all__ = [
-    "InvitationToken",
-    "SASCode",
-    "generate_sas_code_candidates",
-    "generate_sas_codes",
-    "InviteUserConfirmation",
-    "InviteDeviceData",
-    "InviteDeviceConfirmation",
-    "InviteUserData",
-]
```